### PR TITLE
pgsql, oci and mssql schemas cleanup and implemented findUniqueIndexes

### PR DESCRIPTION
**Pull request by nineinchnick**
- removed processing of pgsql unique index with expressions;
- added findUniqueIndexes for oci and mssql;
- cleaned up oci and mssql schema

### CODE REVIEW

**Comment by nineinchnick** [comment]
Remember to apply fixes from #7868 to `findUniqueIndexes`.

File: framework/db/oci/Schema.php [hunk 1]
```diff
@@ -161,19 +161,22 @@ protected function findColumns($table)
...
+
+        $seq_name_sql = <<<SQL
+SELECT ud.referenced_name as sequence_name
+FROM user_dependencies ud
+JOIN user_triggers ut on (ut.trigger_name = ud.name)
+WHERE ut.table_name = :tableName
+AND ud.type='TRIGGER'
+AND ud.referenced_type='SEQUENCE'
+SQL;
+        $sequenceName = $this->db->createCommand($seq_name_sql, [':tableName' => $tableName])->queryScalar();
```
Comment (nineinchnick) [inline framework/db/oci/Schema.php]: Looks like it's missing the fix against `PDO::ATTR_CASE` from #7868.

Comment (samdark) [inline framework/db/oci/Schema.php]: Do you need to apply more changes to it?

Comment (nineinchnick) [inline framework/db/oci/Schema.php]: Oh wait, there's `queryScalar()` so it's fine here. The whole PR is ready.

**Comment by samdark** [comment]
Changelog?

**Comment by nineinchnick** [comment]
@samdark  done. And I keep forgetting about adding [skip CI] to the commit message.

**Comment by samdark** [comment]
Thanks!

**Pull request closed without merging**